engineering_tools/xil-cloud-formation-v0.yaml

```diff
@@ -32,6 +32,10 @@ Parameters:
     Description: The specific carma config which launches xil and platform to make available in the instance
     Type: String
     Default: "usdotfhwastoldev/carma-config:xil-cloud-test"
+  DockerComposeVersion:
```

Comment:
Looks like this doesn't need to be argument for the yaml. Local scope on line 429-430 should be enough

Reply:
Yeah weird syntax error was giving me trouble so I was trying out different approaches. I learned in this aws yaml context that:
- ${VARIABLE} is a parameter defined in the yaml level
- $VARIABLE is a variable for the shell.
Although I did defined the VARIABLE=docker-compose-version, it kept giving me error when I used it as ${VARIABLE} saying it was not resolved.